src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -173,6 +173,21 @@ public Category GetCategory
         }
 
         /// <summary>
+        /// Get Element Type
+        /// </summary>
+        public Element ElementType
+        {
+            get
+            {
+                // Get the active Document
+                Autodesk.Revit.DB.Document document = DocumentManager.Instance.CurrentDBDocument;
+
+                return document.GetElement(this.InternalElement.GetTypeId()).ToDSType(true);
```

Comment:
We should update the description of this property to describe that it could return null. Something like:
> Returns the FamilyType for this Element. Returns null if the Element cannot have a FamilyType assigned.